airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java

```diff
@@ -109,4 +110,18 @@ public static JsonNode deserialize(final String jsonString) {
     return (T) deserialize(serialize(object), object.getClass());
   }
 
+  /**
+   * JsonSchema supports to ways of declaring type. `type: "string"` and `type: ["null", "string"]`.
```

Comment:
I would put his in a `MoreJsonSchema` or smth like that class. It is not a general Json helper